frontend/sw360-portlet/src/main/webapp/html/projects/view.jsp

```diff
@@ -284,36 +294,45 @@
          $('#projectsTable_filter').hide();
          $('#projectsTable_first').hide();
          $('#projectsTable_last').hide();
-     }
+    }
 
 
-     function createUrl_comp(paramId, paramVal) {
+    function createUrl_comp(paramId, paramVal) {
          var portletURL = PortletURL.createURL('<%= PortletURLFactoryUtil.create(request, portletDisplay.getId(), themeDisplay.getPlid(), PortletRequest.RENDER_PHASE) %>')
                  .setParameter('<%=PortalConstants.PAGENAME%>', '<%=PortalConstants.PAGENAME_DETAIL%>').setParameter(paramId, paramVal);
          return portletURL.toString();
-     }
+    }
 
-     function createDetailURLfromProjectId(paramVal) {
+    function createDetailURLfromProjectId(paramVal) {
          return createUrl_comp('<%=PortalConstants.PROJECT_ID%>', paramVal);
-     }
+    }
+
+    function exportExcel() {
```

Comment:
The only duplicated code is the name of the function. The URL is different, the parameters are different.